### DISCUSSION TITLE
spring-circuit-breaker/

### DISCUSSION THREAD

**Discussion by giscus** [Hyeon9mak/Hyeon9mak.github.io#173] [Blog comments]
# spring-circuit-breaker/

circuit breaker 를 처음 접하게 되었을 때 retry 와 비슷한 기능을 하는 것 같아서 자주 헷갈렸다. 특히 @Retryable 과 @Recover 를 조합하다보면 circuit breaker 와 정말 유사한 결과물을 만들 수 있다. 그러나 기술의 컨셉을 이해하고 여러가지 테스트를 해보니 차이가 느껴진다.  

https://hyeon9mak.github.io/spring-circuit-breaker/

**Comment by seovalue** [comment]
요즘 딱 리트라이 찾아보고 있었는데 갓갓

**Comment by Gomding** [comment]
출근길 최고의선택

**Comment by hgs-study** [comment]
퇴근길 최고의선택
